app/views.py

```diff
@@ -83,13 +83,38 @@ def logout():
 @app.route('/showAddScript')
 def showAddScript():
 	if session.get('user'):
-		print session['user']
+		#print (session['user'])
 		return render_template('addScript.html',author = session['user'][0])
 	else:
 		return render_template('error.html',error = 'Unauthorized Access')    
 
 
 
+@app.route('/addScript',methods=['POST'])
+def addScript():
+	try:
+		if session.get('user'):
+			_user = session.get('user')[1]
```

Comment:
Yeah was just saying that this restricts the use of this function to only remote app. If we send user parameter in function we can call this endpoint from any app. But android app hasn't developed to the point to save scripts, so we can change things later.